rave-compiler/src/main/java/com/uber/rave/compiler/CompilerUtils.java

```diff
@@ -47,11 +47,11 @@
 import javax.lang.model.util.Types;
 
 /**
- * Utitlies class for annotations.
+ * Utilities class for annotations.
  */
 final class CompilerUtils {
 
-    private static final List<Class<? extends Annotation>> SUPPORTED_ANNOTATIONS = ImmutableList.of(
+    private static final List<Class<? extends Annotation>> SUPPORT_ANNOTATIONS = ImmutableList.of(
             NonNull.class,
             Nullable.class,
```

Comment:
It would, but if we are making these annotations work without the need to do so, that would be even better. i.e convert all supported annotations to not depend on any artifact from our side as well